### DISCUSSION TITLE
RESPAWN BLOCKED AFTER PLAY AS IND

### DISCUSSION THREAD

**Discussion by Oldman-BR** [Vdauphin/HeartsAndMinds#1642] [Question and answer]
After use a mod that make me play as IND, i did the procedure to change the player side in mission.sqf. After i changed the side in the mission.sqf line 463 (btc_player_side = independent;) and the line 464 (btc_respawn_marker = "respawn_independent";), changed the name of the marker in the editor, the respawn now is blocked. Can you tell me what is the problem please? Thanks.

**Comment by Vdauphin** [comment]
No spam please

**Comment by AshTrayBY** [comment]
Hi, not sure why you considered it to be a spam. having the same issue. changed to btc_player_side = independent; and btc_respawn_marker = "respawn_guerrila"; as it stated at BI wiki. as a result - no respawns available + mission start time drops to 00:15 (it should be 06:15).
Enemy is set to east faction in param.hpp - double checked.

**Reply by Vdauphin** [reply]
Because he creates [one discussion](https://github.com/Vdauphin/HeartsAndMinds/discussions/1642#discussioncomment-15044663), [one issue](https://github.com/Vdauphin/HeartsAndMinds/issues/1641), and comment in [another discussion](https://github.com/Vdauphin/HeartsAndMinds/discussions/1623#discussioncomment-10309423) about the same topic within the same day

**Comment by AshTrayBY** [comment]
disregard. it was not working because one of the mods containing units was not loaded. all is working as intended with btc_player_side = independent; and btc_respawn_marker = "respawn_guerrila"
